### ISSUE TITLE
ssl - big trouble

### ISSUE THREAD

**Issue by vasyapoopken** [canonical/lxd-ui#885]
There is no way to make the ssl certificate work. Added to the trust zone, added in the browser. Does not let into the web-ui. Added an alternative way "lxc remote add username token" - does not work.

**Comment by edlerd** [comment]
Thank you for the report.

Please follow the instructions on the web UI to set up the certificates for your OS and Browser. If the steps don't work, please mention exactly which OS and Browser you are on and which step is not working for you.

You can also find guidance in this [article from our wiki](https://github.com/canonical/lxd-ui/wiki/Authentication-Setup-FAQ).

**Comment by edlerd** [comment]
Closing for lack of response. Please consult the [public forum](https://discourse.ubuntu.com/c/lxd/126) in the future for questions regarding the setup.

**Comment by falconair** [comment]
This user’s frustration is real! 

I’ve wasted several days just trying to get the UI working. I’ve tried almost every combination of creating certificates, using existing ones, trying different browsers. A large number of us just want to use lxd is a way to simulate a large network and learn terraform, kubernetes, docker, Ray, dask, spark, etc.

My simulated cluster is literally on the same laptop as the only user, me!

Please allow this certificate business to be turned off. It is maddening!

**Comment by edlerd** [comment]
Thanks for your message, and sorry you got frustrated.

What exactly was your issue @falconair ? 

As an alternative to certificates, you can start using [OIDC with an identity provider](https://github.com/canonical/lxd-ui/wiki/Setup-oidc-login).

**Comment by edlerd** [comment]
You can also find a [video tutorial](https://youtu.be/wqEH_d8LC1k?feature=shared&t=203) for the certificate setup.

**Comment by falconair** [comment]
@edlerd Thanks for your quick response. The video tutorial showed that I needed to copy the crt file into wsl2. Perhaps it wasn't loading from /mnt/c/...

I am now able to use the UI app via firefox.

I didn't know OIDC method existed, although it isn't quite what I want from my use case. It has lots of steps for someone who just wants to simulate a homelab. Besides, in Step 4 (b?), there is no Actions -> Flows -> Login. Perhaps the menu has changed?

In general, I hope you add a new user story: people who want to simulate networks on their machines. My own goal in using lxd was to replace what I used to do in the old days - setting up a bunch of virtual box instances and playing around with one distributed app or another. No need to worry about security certificates, the way countless web frameworks provide quick and dirty web/application servers which are not supposed to be used i prod.
